src/probability/martingale/convergence.lean

```diff
@@ -21,7 +21,7 @@ an L¹ bounded submartingale.
 * `measure_theory.submartingale.ae_tendsto_limit_process`: the almost everywhere martingale
   convergence theorem: an L¹-bounded submartingale adapted to the filtration `ℱ` converges almost
   everywhere to its limit process.
-* `measure_theory.submartingale.mem_ℒ1_limit_process`: the limit process of an L¹-bounded
+* `measure_theory.submartingale.mem_ℒp_limit_process`: the limit process of an L¹-bounded
   submartingale is integrable.
```

Comment:
Is that lemma really proving `integrable`? or `mem_ℒp`?

Reply:
You're right, I've changed it to talk about Lp now